sensor_msgs/Image[] roi
center[] centers
int64[] bbox_xmin
int64[] bbox_ymin
string[] object_name
